_saas-integrations/recharge/v1/recharge-v1.md

```diff
@@ -0,0 +1,133 @@
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/saas/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#      Page & Formatting     #
+# -------------------------- #
+
+title: ReCharge
+permalink: /integrations/saas/recharge
+keywords: recharge, integration, schema, etl recharge, recharge etl, recharge schema
+layout: singer
+# input: false
+
+key: "recharge-setup"
+
+
+# -------------------------- #
+#         Tap Details        #
+# -------------------------- #
+
+name: "recharge"
+display_name: "ReCharge"
+
+singer: true
+status-url: ""
+
+tap-name: "recharge"
+repo-url: https://github.com/singer-io/tap-recharge
+
+this-version: "1.0"
+
+api: |
+  [ReCharge Payments API](https://developer.rechargepayments.com/){:target="new"}
+
+
+# -------------------------- #
+#       Stitch Details       #
+# -------------------------- #
+
+certified: true 
+
+historical: "1 year"
+frequency: "1 hour"
+tier: "Free"
+
+anchor-scheduling: true
+cron-scheduling: true
+
+extraction-logs: true
+loading-reports: true
+
+table-selection: true
+column-selection: true
+
+# attribution-window: "# days"
+# attribution-is-configurable: 
+
+# setup-name: ""
+
+
+# -------------------------- #
+#      Feature Summary       #
+# -------------------------- #
+
+feature-summary: |
+  Stitch's {{ integration.display_name }} integration replicates data using the {{ integration.api | flatify | strip }}. To access the {{ integration.display_name }} API documentation, you will need [your API key](#obtain-api-key). Refer to the [Schema](#schema) section for a list of objects available for replication.
+
+
+
+# -------------------------- #
+#      Incompatibilities     #
+# -------------------------- #
+
+## uncomment section below if integration is compatible with any Stitch destinations
+## if incompatible with multiple destinations, create a section for each destination
+
+## incompatible:
+  ## [redshift]: "always,sometimes,never"
+  ## reason: "copy" 
+
+# -------------------------- #
+#      Setup Instructions    #
+# -------------------------- #
+
+requirements-list:
+  - item: "**A Shopify Account**. You need this to access your {{ integration.display_name }} information."
+
+setup-steps:
+  - title: "Obtain Your {{ integration.display_name }} API Key"
+    anchor: "obtain-api-key"
+    content: |
+      1. Log into your Shopify account, select `Apps`, and then click on your {{ integration.display_name }} application.
+         {% include layout/image.html file="/integrations/recharge-apps-screen.png" alt="Shopify Apps page." enlarge=true max-width="550" %}
+      {:start="2"}
+      2. In the application, click on the `EXPLORE {{ integration.display_name }}` option.
+         {% include layout/image.html file="/integrations/recharge-explore-button.png" alt="EXPLORE ReCharge button." enlarge=true max-width="250" %}
```

Comment:
I think in this case, less screenshots are probably more. If any of this changes on ReCharge's end, these screenshots will need to be updated. I also wasn't entirely sure *where* to find this on the page, and it took me a minute to find it in Shopify.

What if you added some more detail about where this link is located instead of using a screen shot? Something like:

```
2. In the application, click the **EXPLORE {{ integration.display_name }}** located near the top right corner of the page.
```